testing/trino-product-tests/src/main/java/io/trino/tests/product/hive/TestHiveTransactionalTable.java

```diff
@@ -1426,7 +1426,7 @@ public void testAcidUpdateWithSubqueryAssignment()
 
     @Flaky(issue = ERROR_COMMITTING_WRITE_TO_HIVE_ISSUE, match = ERROR_COMMITTING_WRITE_TO_HIVE_MATCH)
     @Test(groups = HIVE_TRANSACTIONAL, timeOut = TEST_TIMEOUT)
-    public void testInsertDeletUpdateWithPrestoAndHive()
+    public void testInsertDeleteUpdateWithPrestoAndHive()
```

Comment:
withTrino :)

Reply:
good catch! there are many other uses of word "presto" in this class. And this doesn't qualify as a typo, so would need yet another commit. next time